search/meilisearch.py

```diff
@@ -177,6 +186,10 @@ def search(
         )
         if log_search_params:
             logger.info("Search query: opt_params=%s", opt_params)
+        # Apply sort parameters from settings when no sort criteria specified in request
+        if "sort" not in opt_params:
+            features = getattr(settings, 'FEATURES', {})
+            opt_params["sort"] = features.get('MEILISEARCH_DISCOVERY_SORT_PARAMETERS', [])
```

Comment:
Can we make this generalized? This seems to be specific to MEILISEARCH only; we should include this for both search engines.